cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -937,6 +937,7 @@ runQueryStakeAddressInfoCmd
 data StakeAddressInfoData = StakeAddressInfoData
```

Comment:
I think this type is now deserving of a haddock that outlines what the various records correspond to.